src/style.css

```diff
@@ -1,89 +1,134 @@
-/* reset default styles */
-* {
-  box-sizing: border-box;
+html,
+body {
   margin: 0;
   padding: 0;
+  height: 100%;
+  width: 100%;
+  overflow-x: hidden;
+  font-size: 16px;
+  line-height: 1.5;
 }
 
-/* basic styles */
-body {
-  font-family: Arial, sans-serif;
-  background-color: #f5f5f5;
-  color: #333;
+header {
+  margin-left: 10%;
 }
 
-header {
-  background-color: #333;
-  color: #fff;
-  padding: 1em;
-  text-align: center;
-  margin: 30px;
+.title {
+  color: black;
+  margin-left: 20px;
+}
 
-  /* text-decoration: none; */
-  list-style: none;
+.flexo {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: auto;
+  position: relative;
+  margin: 0 15%;
 }
 
 .top {
-  border: 2px solid black;
+  padding: 10px;
+  width: 60%;
 }
 
-li {
-  list-style: none;
+.recentScores {
+  color: black;
+  margin: 10px;
+  font-size: 30px;
+}
+
+.btn-refresh {
+  color: #202020;
+  background-color: #fff;
+  text-align: center;
   text-decoration: none;
+  display: inline-block;
+  font-size: 16px;
+  margin: 4px 2px;
+  cursor: pointer;
+  border: 1px solid black;
+  box-shadow: 2px 2px black;
+}
+
+.top-heading {
+  display: flex;
+  flex-direction: row;
+  justify-content: space-between;
+  align-items: center;
+  width: auto;
 }
 
-main {
-  max-width: 800px;
-  margin: 0 auto;
-  padding: 1em;
+#leaderboard {
+  border: 2px solid black;
 }
 
 form {
-  border: 1px solid #ccc;
-  padding: 1em;
-  margin-bottom: 1em;
-  background-color: #fff;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  width: 50%;
+  margin-left: 10%;
+}
+
+form input,
+button {
+  width: 45%;
+  padding: 10px;
+  margin: 5px;
+  border: 2px solid black;
+  box-shadow: 1px 1px black;
 }
 
-fieldset {
-  border: none;
+ul {
+  list-style: none;
   padding: 0;
+  margin: 0;
+  width: 100%;
 }
 
-legend {
-  font-size: 1.2em;
-  font-weight: bold;
+.header {
+  display: flex;
+
+  /* justify-content: space-between; */
```

Comment:
- [ ] [OPTIONAL] Just like I we agreed earlier that codes that are no longer in use should best be removed to avoid messy workspace or codes that are difficult to read, with increased probability of bug introduction. I suggest that you remove this commented code too, or what do you think? :thought_balloon: